### DISCUSSION TITLE
SSL Certificate won't generate

### DISCUSSION THREAD

**Discussion by siddharthd0** [vercel/community#2754] [Help] [answered]
### Summary

I'm not sure what the problem is, but it has been 24 hours and the SSL certificate won't work.
<img width="1030" alt="Screen Shot 2023-06-06 at 12 17 14 PM" src="https://github.com/vercel/community/assets/96356209/c56ee9b8-79bb-4528-88f4-9596082592d8">

### Steps to Reproduce

I included a screenshot above but I'm not sure how one can reproduce this problem.

**Comment by swarnava** [comment, marked as answer]
It looks like we are having trouble to fetch DNS record for your domain:

- https://letsdebug.net/www.siddharthduggal.com/1508564
- https://dnschecker.org/#CNAME/www.siddharthduggal.com

Please contact to your domain registrar for more information on this.
